packages/manager/modules/cloud-universe-components/src/index.js

```diff
@@ -1,30 +1,21 @@
 import angular from 'angular';
 
-import cucAutoComplete from './autocomplete';
 import cucAdvancedOptions from './cui/advanced-options';
 import cucAutofocus from './autofocus';
 import cucAutoSelect from './autoselect';
 import cucBytes from './bytes';
-import cucClickEnterOnKeypress from './clickEnterOnKeypress';
 import cucConfig from './config';
-import cucConsumption from './consumption';
 import cucContracts from './contracts';
-import cucCurrency from './currency';
 import cucFeatureAvailability from './featureAvailability';
-import cucFlavor from './flavor';
 import cucHelper from './helper';
-import cucHighlightedElement from './highlightedElement';
 import cucMessage from './message';
 import cucMomentFormat from './moment';
-import cucMonitoring from './monitoring';
 import cucNavigation from './navigation';
 import cucOrderedHash from './orderedHash';
 import cucPoll from './poll';
 import cucPrice from './price';
 import cucRegion from './region';
 import cucProducts from './products';
-import cucSmoothScrollHere from './smoothScrollHere';
-import cucSpaceMeter from './space-meter';
 import cucSshKeyMin from './sshKeyMin';
```

Comment:
Looks like there is the same filter in the dedicated app.  We have moved it to dedicated universe components.

https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/account/user/components/filters/sshkeyMin/sshkeyMin.js

Reply:
yes but it's also used in cloud app, how did you handle the case ?
(https://github.com/ovh/manager/blob/master/packages/manager/apps/cloud/client/app/cloud/project/compute/ssh/cloud-project-compute-ssh.html#L331)

Reply:
I moved it to dedicated universe componets. We need to revisit to consolidate this.